src/shared/components/oncoprint/geneticrules.ts

```diff
@@ -256,25 +256,45 @@ const non_mutation_rule_params: GeneticAlterationRuleParams = {
             // Red rectangle for amplification
             amp: {
                 shapes: [shapeBank[ShapeId.ampRectangle]],
-                legend_label: 'Amplification',
+                legend_label: 'Amplification (unknown significance)',
+                legend_order: AMP_LEGEND_ORDER,
+            },
+            amp_rec: {
+                shapes: [shapeBank[ShapeId.ampRectangle]],
+                legend_label: 'Amplification (putative driver)',
                 legend_order: AMP_LEGEND_ORDER,
```

Comment:
I think we should put the driver legend items consistently first

Reply:
I updated the order.

Reply:
it looks the same to me?

Reply:
@adamabeshouse Hmm then I misunderstand you. How should they be ordered? At present the order is:
```
            amp_rec: {
                shapes: [shapeBank[ShapeId.ampRectangle]],
                legend_label: 'Amplification (putative driver)',
                legend_order: AMP_LEGEND_ORDER,
            },
            // Red rectangle for amplification
            amp: {
                shapes: [shapeBank[ShapeId.ampRectangle]],
                legend_label: 'Amplification (unknown significance)',
                legend_order: AMP_LEGEND_ORDER,
            },
            gain_rec: {
                shapes: [shapeBank[ShapeId.gainRectangle]],
                legend_label: 'Gain (putative driver)',
                legend_order: GAIN_LEGEND_ORDER,
            },
            // Light red rectangle for gain
            gain: {
                shapes: [shapeBank[ShapeId.gainRectangle]],
                legend_label: 'Gain (unknown significance)',
                legend_order: GAIN_LEGEND_ORDER,
            },
            homdel_rec: {
                shapes: [shapeBank[ShapeId.homdelRectangle]],
                legend_label: 'Deep Deletion (putative driver)',
                legend_order: HOMDEL_LEGEND_ORDER,
            },
            // Blue rectangle for deep deletion
            homdel: {
                shapes: [shapeBank[ShapeId.homdelRectangle]],
                legend_label: 'Deep Deletion (unknown significance)',
                legend_order: HOMDEL_LEGEND_ORDER,
            },
            hetloss_rec: {
                shapes: [shapeBank[ShapeId.hetlossRectangle]],
                legend_label: 'Shallow Deletion (putative driver)',
                legend_order: HETLOSS_LEGEND_ORDER,
            },
            // Light blue rectangle for shallow deletion
            hetloss: {
                shapes: [shapeBank[ShapeId.hetlossRectangle]],
                legend_label: 'Shallow Deletion (unknown significance)',
                legend_order: HETLOSS_LEGEND_ORDER,
            },
```

Reply:
the order is determined by the `legend_order` member, not by the order in the object literal. so as it is now, for example, `hetloss_rec` has the same order as `hetloss`, which means that (I think) depending on random fluctuations in the data we could see them appearing before or after each other. better to keep it set so its stable

Reply:
So, how is this different from this section?
```

                    missense_rec: {
                        shapes: [
                            shapeBank[ShapeId.missenseMutationDriverRectangle],
                        ],
                        legend_label: 'Missense Mutation (putative driver)',
                        legend_order: MUTATION_LEGEND_ORDER,
                    },
                    missense: {
                        shapes: [
                            shapeBank[ShapeId.missenseMutationVUSRectangle],
                        ],
                        legend_label:
                            'Missense Mutation (unknown significance)',
                        legend_order: MUTATION_LEGEND_ORDER,
                    },
```

Reply:
you're right. I guess the order is arbitrary right now for mutations. that shouldn't be that way but I guess it's probably fine to just go as-is for this PR